<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiHttpWebRequests</class>
 <widget class="QWidget" name="UiHttpWebRequests">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HTTP Web Requests Composer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_TOP_DOWN">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter_COMPOSER_HISTORY">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout_ComposerGroup">
        <item row="1" column="0" colspan="2">
         <widget class="QSplitter" name="splitter_ComposerRequestAndData">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QPlainTextEdit" name="plainTextEdit_ComposerRawHeaders">
           <property name="plainText">
            <string>User-Agent: Crypto Pocket Knife 1.1</string>
           </property>
           <property name="placeholderText">
            <string>GET /index.php HTTP/1.1</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_Composer_Submit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Submit</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_Composer_TargetHost">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>https://sectelecom.net/test</string>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>https://test.com/aaa</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="lineEdit_Composer_HttpMethod">
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>GET</string>
          </property>
          <property name="placeholderText">
           <string>GET</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="formLayoutWidget">
       <layout class="QGridLayout" name="gridLayout_History">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>History</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string>Response Code</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Method</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>URL</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_REQUEST_RESPONSE">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_REQUEST">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>REQUEST</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget_REQUEST">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="tab_XML_3">
           <attribute name="title">
            <string>XML</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_JSON_3">
           <attribute name="title">
            <string>JSON</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_RawOutput_3">
           <attribute name="title">
            <string>RAW Output</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QPlainTextEdit" name="plainTextEdit_OutputRAW_REQUEST">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_RESPONSE">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>RESPONSE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget_RESPONSE">
          <property name="currentIndex">
           <number>3</number>
          </property>
          <widget class="QWidget" name="tab_WebView_2">
           <attribute name="title">
            <string>Web view</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_XML_2">
           <attribute name="title">
            <string>XML</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_JSON_2">
           <attribute name="title">
            <string>JSON</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_RawOutput_2">
           <attribute name="title">
            <string>RAW Output</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QPlainTextEdit" name="plainTextEdit_OutputRAW_RESPONSE">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
